next/backend/forms/test/schema.json

```diff
@@ -117,31 +117,49 @@
         }
       }
     },
-    {
+     {
       "properties": {
-        "chooseOneOf": {
-          "title": "Choose one of radio buttons",
+        "checkBoxes": {
+          "title": "Checkboxes",
           "properties": {
```

Comment:
we want both to show the radio buttons and checkboxes in the example schema - please add checkboxes instead of replacing